source/help/cloud.md

```diff
@@ -0,0 +1,9 @@
+Served from the cloud
+===============
+
+For approximately the last 20 years, arXiv's services have been hosted on a set of services hosted at Cornell's central IT facility in Ithaca New York. In recent years, we've approached the limits of what a University IT facility can offer, and have thus begun to shift our services to a public cloud vendor.
+
+We are doing this process little by little, starting with the browsing activities &ndash; serving the home page, listings, abstracts, and most importantly, article downloads (PDF and other formats). The portions of the arXiv site that enable users to change things (account creation and mainteance, submiting and updating articles) will transition later.
```

Comment:
Simplify:

The transition will happen later for account creation and maintenance, submitting and updating articles, and other pages.

Also check spelling of maintenance and submitting.

Reply:
Maybe think ahead to search; readers may have a different understanding of what includes browsing; and put the most important thing first:

> starting with the browsing activities &ndash; serving the home page, listings, abstracts, and most importantly, article downloads (PDF and other formats).

starting with PDFs, abstracts, listings, and search.